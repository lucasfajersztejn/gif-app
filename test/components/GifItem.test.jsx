import { render } from "@testing-library/react";
import { GifItem } from "../../src/components/GifItem"

describe('test on <GifItem />', () => {

  const title = 'title is needed'
  const url = `https://api.giphy.com/v1/gifs/search?api_key=xM25jB2FxNgSRzRmX4IM5cuc69lUFBIT&q=DragonBall&limit=10`;

  test('title and url are needed', () => {
    const { container } = render( <GifItem title={title} url={url}/> );
    expect( container ).toMatchSnapshot();    
  })
})