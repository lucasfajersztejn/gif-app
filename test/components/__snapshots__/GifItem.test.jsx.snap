// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`test on <GifItem /> title and url are needed 1`] = `
<div>
  <div
    class="card"
  >
    <img
      alt="title is needed"
      src="https://api.giphy.com/v1/gifs/search?api_key=xM25jB2FxNgSRzRmX4IM5cuc69lUFBIT&q=DragonBall&limit=10"
    />
    <p>
      title is needed
    </p>
  </div>
</div>
`;
